### PR TITLE
Added basic description for GCC,Make etc

### PR DESCRIPTION
**Pull request by parth93QA**
Add Content For DevOps Roadmap for 106-compiling-apps.md

### CODE REVIEW

File: content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/106-compiling-apps.md [hunk 1]
```diff
@@ -1 +1,43 @@
-# Compiling apps
\ No newline at end of file
+# Compiling apps
```
Comment (kamranahmedse) [inline content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/106-compiling-apps.md:1]:
```suggestion
# Compiling Apps
```

File: content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/106-compiling-apps.md [hunk 2]
```diff
@@ -1 +1,43 @@
...
+
+#sbt
+
+sbt is an open-source build tool for Scala and Java projects, similar to Apache's Maven and Ant. Its main features are: Native support for compiling Scala code and integrating with many Scala test frameworks. Continuous compilation, testing, and deployment.
+
+<ResourceGroupTitle>Free Content</ResourceGroupTitle>
+<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.scala-sbt.org/1.x/docs/'>Sbt Documentation</BadgeLink>
+<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.scala-sbt.org/1.x/docs/sbt-by-example.html'>Sbt By Example</BadgeLink>
+
+#gradle
```
Comment (kamranahmedse) [inline content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/106-compiling-apps.md:34]:
```suggestion
# gradle
```

File: content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/106-compiling-apps.md [hunk 3]
```diff
@@ -1 +1,43 @@
...
+Make builds the program from a file called the makefile, which lists each of the non-source files and how to compute it from other files. When you write a program, you should write a makefile for it, so that it is possible to use Make to build and install the program.
+
+"What is make used for?" Make enables the end user to build and install your package without knowing the details of how that is done -- because these details are recorded in the makefile that you supply.
+
+<ResourceGroupTitle>Free Content</ResourceGroupTitle>
+<BadgeLink colorScheme='yellow' badgeText='Read' href='https://makefiletutorial.com'>Makefile Tutorial</BadgeLink>
+<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.gnu.org/software/make/manual/'>Documentation for make</BadgeLink>
+<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.cs.swarthmore.edu/~newhall/unixhelp/howto_makefiles.html'>Using Make and writing Makefiles</BadgeLink>
+
+#sbt
```
Comment (kamranahmedse) [inline content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/106-compiling-apps.md:26]:
```suggestion
# sbt
```

File: content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/106-compiling-apps.md [hunk 4]
```diff
@@ -1 +1,43 @@
...
+
+"What is GCC used for?" GCC is a toolchain that compiles code, links it with any library dependencies, converts that code to assembly, and then prepares executable files.It is responsible for the conversion of the “high level” source code in the respective language and ensuring that it is semantically valid, performing well formed optimizations, and converting it to assembly code (which is then handed off to the assembler).
+
+<ResourceGroupTitle>Free Content</ResourceGroupTitle>
+<BadgeLink colorScheme='yellow' badgeText='Read' href='https://courses.cs.washington.edu/courses/cse451/99wi/Section/gccintro.html'>Intro to GCC</BadgeLink>
+<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.javatpoint.com/gcc-linux'>GCC Linux</BadgeLink>
+<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.geeksforgeeks.org/gcc-command-in-linux-with-examples/'>GCC Commands</BadgeLink>
+
+#make
```
Comment (kamranahmedse) [inline content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/106-compiling-apps.md:14]:
```suggestion
# make
```

**Review by kamranahmedse — APPROVED** [review]

**Comment by parth93QA** [comment]
Thank You @kamranahmedse

**Pull request merged**